src/main/java/uk/gov/pay/api/validation/URLValidator.java

```diff
@@ -6,7 +6,7 @@
 
 public enum URLValidator {
     SECURITY_ENABLED {
-        private final UrlValidator URL_VALIDATOR = new UrlValidator(new String[]{"https"});
+        private final UrlValidator URL_VALIDATOR = new UrlValidator(new String[]{"https"}, ALLOW_LOCAL_URLS);
```

Comment:
Is this a good thing? This means that If someone sends us a `localhost`  `return_url` we accept it?